packages/integration-sdk-core/jest.config.js

```diff
@@ -1,4 +1,4 @@
 module.exports = {
   ...require('../../jest.config.base'),
-  setupFilesAfterEnv: ['./jest.setup.ts'],
+  //setupFilesAfterEnv: ['./jest.setup.ts'],
```

Comment:
why is this line commented?\

Reply:
The file doesn't actually exist

Reply:
@sebassdc

Reply:
@labs-dlugo why dont we delete the line then?

Reply:
ok